azure-pipelines-dev.yml

```diff
@@ -373,8 +373,16 @@ stages:
               pip install pytest-cov pytest-azurepipelines
 
               # Run pytest
-              pytest tests --ignore 'tests/cli' --ignore 'contrib/cli/tests' --ignore 'tests/integration/usage_statistics' \
-                --mysql --napoleon-docstrings --junitxml=junit/test-results.xml --cov=. --cov-report=xml --cov-report=html
+              pytest tests/test_definitions tests/expectations \
```

Comment:
Per some research I did, I think these two directories encapsulate all of our backend-specific logic. Instead of retesting various GX components like RBP, Validator, and DataContext, let's narrow the scope of our testing so we only cover what is specific to Trino/MSSQL/MySQL/etc.